pkg/filter/filter.go

```diff
@@ -138,16 +136,63 @@ func (f *filter) ShouldIgnoreDDLEvent(ddl *model.DDLEvent) (bool, error) {
 // If a ddl is discarded, it will not be applied to cdc's schema storage
 // and sent to downstream.
 func (f *filter) ShouldDiscardDDL(ddlType timodel.ActionType) bool {
-	if !shouldDiscardByBuiltInDDLAllowlist(ddlType) {
+	/* The following DDL will be filter:
+	ActionAddForeignKey                 ActionType = 9
+	ActionDropForeignKey                ActionType = 10
+	ActionRebaseAutoID                  ActionType = 13
+	ActionShardRowID                    ActionType = 16
+	ActionLockTable                     ActionType = 27
+	ActionUnlockTable                   ActionType = 28
+	ActionRepairTable                   ActionType = 29
+	ActionSetTiFlashReplica             ActionType = 30
+	ActionUpdateTiFlashReplicaStatus    ActionType = 31
+	ActionCreateSequence                ActionType = 34
+	ActionAlterSequence                 ActionType = 35
+	ActionDropSequence                  ActionType = 36
+	ActionModifyTableAutoIdCache        ActionType = 39
+	ActionRebaseAutoRandomBase          ActionType = 40
+	ActionAlterIndexVisibility          ActionType = 41
+	ActionExchangeTablePartition        ActionType = 42
+	ActionAddCheckConstraint            ActionType = 43
+	ActionDropCheckConstraint           ActionType = 44
+	ActionAlterCheckConstraint          ActionType = 45
+	ActionAlterTableAlterPartition      ActionType = 46
+
+	... Any Action which of value is greater than 46 ...
+	*/
+	switch ddlType {
```

Comment:
Could you please add a slice or map name `ddlAllowList` to contains all DDLs that allow by cdc?